ci/Makefile

```diff
@@ -2,7 +2,17 @@
 .PHONY: setup-conda-env build-hail-ci push-hail-ci test-locally
 .PHONY: run-service deploy
 
-PROJECT = $(shell gcloud config get-value project)
+PROJECT=${shell gcloud config get-value project}
```

Comment:
I think we've tended to prefer spaces around equals signs in Make. The [docs use this convention](https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html). It doesn't much matter to me but I prefer consistency across our codebase.

Reply:
Got it. I mostly did no space because it was the shell convention. I'll add spaces if that's the standard.

Reply:
Depending on where spaces appear, assignment either works or does not work (shell or makefile context):

https://stackoverflow.com/questions/42157800/makefile-variable-assignment-error-in-echo

Should we remove one source of error by standardizing on no space assignments? 

In last commit I left spaces where they are ok

Reply:
That SO question and its answer are a bit misguided. The recipe of a make target is a series of lines. Each line is executed independently in a shell (by default `/bin/sh`, controlled by the make variable `SHELL`). That first line is erring because `/bin/sh` thinks you're executing a command `i` with the arguments `=` and `1`. Moreover, that variable is inaccessible to subsequent lines of the recipe (each line of a recipe gets an independent shell process).

I have no preference aesthetically for or against braces and spaces around the assignment operator, but I think we should stick to the Makefile conventions (i.e. the style used in the GNU Make docs).

FWIW, make variables are semantically different (especially wrt when they're defined) from shell variables and it's good to have the intention clear from the syntax. Specifically, if I ever see `${FOO}` in a Makefile, I immediately suspect the writer made a mistake and intended to write `$${FOO}` (`$$` is the make escape code to produce a single dollar sign) thus producing a shell variable reference (which is evaluated at make recipe execution time, not Makefile parse time).

Reply:
Valid points. I was looking at the last SO comment, which provides a similar explanation. Simple rules aren't enough, make syntax is inconsistent, and depends on hidden rules resulting in apparently-magical behavior:

Legal:
`phony-target: IMAGE = something`

Not:
```
phony-target:
   IMAGE = something some-command
```

Legal (despite executing in the shell)
```
phony-target:
   $(SomeVar}
```

Not (in /binsh)
```sh
 $(SomeVar}
```

While possible to get right, these kinds of implementation differences should be hidden from the user. Imagine if you wrote Hail such that the user was required to know both the entire surface of Python and Java, or to have vastly different syntax in local or distributed modes.